packages/ui/components/form/select/Select.tsx

```diff
@@ -51,7 +51,7 @@ export const Select = <
         dropdownIndicator: () => "text-default",
         control: (state) =>
           classNames(
-            "bg-default border-default !min-h-9 text-sm leading-4 placeholder:text-sm placeholder:font-normal focus-within:ring-2 focus-within:ring-emphasis hover:border-emphasis rounded-md border ",
+            "bg-default hover:border-emphasis border-default !min-h-9 text-sm leading-4 placeholder:text-sm placeholder:font-normal focus-within:border-neutral-300 focus-within:outline-none focus-within:ring-2 focus-within:ring-neutral-800 focus-within:ring-offset-1  rounded-md border ",
```

Comment:
I dont think we need neutral here.

```suggestion
            "bg-default hover:border-emphasis border-default !min-h-9 text-sm leading-4 placeholder:text-sm placeholder:font-normal focus-within:border-default focus-within:outline-none focus-within:ring-2 focus-within:ring-emphasis focus-within:ring-offset-1  rounded-md border ",
```

Reply:
We may not even need `focus-within:border-emphasis` since we are making use of ring. Might be worth checking how its handled on other Inputs

Reply:
@sean-brydon `ring-emphasis` is not working. ring looks grey while applying it 🤔